src/pages/auth/login/Login.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as Yup from 'yup';
+import { Link as LinkRoute, useNavigate } from 'react-router-dom';
+import LoadingButton from '@mui/lab/LoadingButton';
+import { TextField, Button, Paper, Box, Grid, Typography } from '@mui/material';
+import useAuth from '../../../hooks/useAuth';
+
+interface IFormInputs {
+  email: string;
+  password: string;
+}
+
+const Login = () => {
+  return <LoginComponent />;
+};
+
+export default Login;
+
+const LoginComponent = () => {
+  const [loading, setLoading] = useState(false);
+  const [message, setMessage] = useState('');
+  const navigate = useNavigate();
+  const { signIn } = useAuth();
+  const onSubmit = async (data: IFormInputs) => {
+    setLoading(true);
+    setMessage('');
+    try {
+      await signIn(data.email, data.password);
+      navigate('/');
+    } catch (e: any) {
```

Comment:
```suggestion
    } catch (e) {
```

Hàm catch là 1 trường hợp đặc biệt trong Typescript, ko cần ghi type cho exception